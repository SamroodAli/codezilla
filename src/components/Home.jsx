import {
  Display1,
  Display3,
} from 'baseui/typography';
import { Block } from 'baseui/block';

const Home = () => (
  <Block display="grid" justifyContent="center" margin="10rem">
    <Display1>Welcome to CodeZilla</Display1>
    <Display3>Codezilla has got everything you need.</Display3>
  </Block>
);

export default Home;
