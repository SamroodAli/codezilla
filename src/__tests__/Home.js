import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';
import Home from '../components/Home';

describe('Testing Home', () => {
  it('Home is defined', async () => {
    const component = render(<Home />);
    expect(component).toBeDefined();
  });

  it('Home snapshot testing', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
