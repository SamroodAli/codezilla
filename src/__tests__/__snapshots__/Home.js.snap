// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Home with redux Home snapshot testing 1`] = `
<div
  className=""
  data-baseweb="block"
>
  <div
    className=""
    data-baseweb="typo-displaylarge"
  >
    Welcome to CodeZilla
  </div>
  <div
    className=""
    data-baseweb="typo-displaysmall"
  >
    Codezilla has got everything you need.
  </div>
</div>
`;
